examples/pytorch/arma/README.md

```diff
@@ -0,0 +1,78 @@
+# DGL Implementation of ARMA
+
+This DGL example implements the GNN model proposed in the paper [Graph Neural Networks with convolutional ARMA filters](https://arxiv.org/abs/1901.01343). For the original implementation, see [here](https://pytorch-geometric.readthedocs.io/en/latest/modules/nn.html#torch_geometric.nn.conv.ARMAConv).
+
+Contributor: [xnuohz](https://github.com/xnuohz)
+
+### Requirements
+The codebase is implemented in Python 3.6. For version requirement of packages, see below.
+
+```
+dgl 0.6a210202
```

Comment:
"0.6a210202" is not a formal version. Either you specify a formal version or just not mention it.

Reply:
Fine, the version will be removed.